File: Cpp/Elevator Distance.cpp
#include <vector>
#include <iostream>
using namespace std ; 

int elevator_distance(vector<int> array) {
  
    int result = 0 ; 
    // array = [2, 5 , 3, 4]
    for(int i = 0 ; i<array.size()-1 ; i++)
    {
        result +=  abs(array[i]-array[i+1]) ; 
    }   
    return result ;
    
}

File: python/Elevator Distance.py
# // https://www.codewars.com/kata/59f061773e532d0c87000d16/train/python


def elevator_distance(array): 
    result = 0  
    for i in range(len(array) - 1):  
        result += abs(array[i] - array[i+1]) 
    return result
